docs/developer-guide/unit-testing.md

```diff
@@ -235,9 +191,9 @@ class Test_iam_password_policy_uppercase:
         expiration=True,
     )
 
-    # We set a mocked audit_info for AWS not to share the same audit state
-    # between checks
-    current_audit_info = self.set_mocked_audit_info()
+    # We set a mocked aws_provider to unify providers, this way will share provider info
+    # between tests
```

Comment:
We don't want to share provider info between tests, the point is to isolate each test not to step on other test's configuration.

Reply:
My bad, missed it